test/utils/image/manifest.go

```diff
@@ -294,7 +300,22 @@ func (i *Config) GetE2EImage() string {
 	return fmt.Sprintf("%s/%s:%s", i.registry, i.name, i.version)
```

Comment:
this print

fmt.Sprintf("%s/%s:%s", i.registry, i.name, i.version)

was repeated 4 times in this code. any way to reduce these duplication?

Reply:
the dup-ed printf is fine since they are oneline string building.
if your concerns is why we added two new methods ( so the dup gets from 2 to 4 ), this is due to the format of the image differs from container to vms. i.e. for containers, they are site/name/version, for the cirros vm image, it is site/version/name

Reply:
my main concern is the wording duplication